action.yml

```diff
@@ -36,13 +36,38 @@ runs:
         VERSION: latest
       run: python -m pip install -r ${{ github.action_path }}/requirements.txt git+https://github.com/wandb/client.git@$VERSION
```

Comment:
BTW, what is `$VERSION` here? It should be pointed at the latest release (and not master), right?

Reply:
yep! on line 36 it's set to `latest`